### PR TITLE
added padding in the header section

### PR DESCRIPTION
**Pull request by dev-keshav**
I have added some CSS in header section and also on the card to look good.

### Summary by CodeRabbit

**Style**: Improved User Interface on the Home Page
- Enhanced readability and visual appeal by adding padding to the Typography component.
- Improved layout and spacing by adding margin to the Grid component. These changes aim to provide a more comfortable and engaging user experience.

### CODE REVIEW

File: frontend/src/components/Pages/Home/index.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const Home = () => {
 
     return (
         <React.Fragment>
-            <Typography variant='h5' align='center' component='h2' style={{ margin: '20px' }}>Make your own personalised style for the Quotes</Typography>
+            <Typography variant='h5' align='center' component='h2' style={{ margin: '20px', padding: '25px' }}>Make your own personalised style for the Quotes</Typography>
```
Comment (shravan20) [inline frontend/src/components/Pages/Home/index.js:17]: LGTM, can you revert the package-lock.json?

**Review by shravan20 — COMMENTED** [review]
Thank you for your time! Your support and contribution matters a lot! :) 

Looking forward for more further contributions too.

I would be relaunching this again by end of the month and should be available for everyone as before.

File: frontend/src/components/Pages/Home/index.js (hunk 1 above)
Comment (dev-keshav) [reply frontend/src/components/Pages/Home/index.js:17]: sure

Comment (dev-keshav) [reply frontend/src/components/Pages/Home/index.js:17]: It is done

**Pull request merged**